Monika After Story/game/script-holidays.rpy

```diff
@@ -92,6 +181,21 @@ init -1 python:
         """
         return mas_isO31(_date) or mas_isF14(_date) or mas_isD25Outfit(_date)
 
+    def mas_isFirstSeshPast(_date):
```

Comment:
this should be moved to `definitions` next to the other session functions (`mas_getFirstSesh`)